pkg/api/v1/verification_types.go

```diff
@@ -45,17 +45,21 @@ const (
 	ReportTypeClinical = "likely"
 	// ReportTypeNegative is allowed by the verification flow. These keys are not saved in the system.
 	ReportTypeNegative = "negative"
+	// ReportTypeSelfReport indicates to set ReportType.SELF_REPORT
+	ReportTypeSelfReport = "selfreport"
 
 	TransmissionRiskUnknown           = 0
 	TransmissionRiskConfirmedStandard = 2
 	TransmissionRiskClinical          = 4
+	TransmissionRiskSelfReport        = 5
```

Comment:
It's cool that we left room for this, but I worry about how clients are handling TransmissionRisk. If they have client-side logic like risk > 4, this could mess with that. I don't have a good solution for it, but it could break existing risk models in obscure ways.

Reply:
https://developers.google.com/android/exposure-notifications/exposure-notifications-api#data-structures

Reply:
I know we documented it. We also know that people don't always follow the documentation.